JavaFXUserInterface/src/main/java/controllers/AssetTypeController.java

```diff
@@ -46,26 +49,35 @@
     @FXML
     private TableColumn<AssetTypeList, Double> columnFailed;
 
+    @FXML
+    private FlowPane assetsTypeThumbPane;
+
     //Configure buttons
     @FXML
     private Button addTypeBtn;
 
     private UIUtilities uiUtilities;
+    private ArrayList<AssetType> assetTypes;
+    private AssetTypeDAOImpl assetTypeDOA;
 
     @Override
     public void initialize(URL url, ResourceBundle resourceBundle) {
         uiUtilities = new UIUtilities();
+        assetTypeDOA = new AssetTypeDAOImpl();
+        assetTypes = assetTypeDOA.getAssetTypeList();
         attachEvents();
-        fillAssetTypeTable();
+        ObservableList<AssetTypeList> assetTypeList = getAssetList();
```

Comment:
I know this wasn't doen in your issue, but could you refactor (rename) the getAssetList() to be getAssetTypeList()?

Reply:
done